### PR TITLE
feature/docs v2

### PR DESCRIPTION
**Pull request by JohannesRudolph**
- chore: enable tflint on the kit
- feat: initial implementation of the docs-v2 approach
- docs: improve sidebar positioning of azure building blocks
- docs: improve Azure platform module documentation
- docs: remove AWS for now as we don't have tfstate management solved
- chore: update docs build to use collie docs v2
- chore: make build workflows ready for multi-platform deployment

### CODE REVIEW

File: kit/foundation/docs/main.tf [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+#   guides = try(data.terraform_remote_state.docs["meshstack"].outputs.documentation_guides_md, {})
+# }
+
+# resource "local_file" "meshstack_guides" {
+#   depends_on = [null_resource.copy_template]
+#   for_each   = local.guides
+
+#   filename = "${var.output_dir}/docs/meshstack/guides/${each.key}.md"
+#   content  = each.value
+# }
```
Comment (florianow) [inline kit/foundation/docs/main.tf:160]: do we need it when it is greyed out?

File: .github/workflows/build-workflow.yml [hunk 2]
```diff
@@ -21,52 +15,57 @@ jobs:
...
-        run: collie info
+      # TODO GCP
+      # - name: Setup gcloud auth
+      #   if: ${{ inputs.platform == 'gcp' }}
+      #   id: 'auth'
+      #   uses: 'google-github-actions/auth@v2'
+      #   with:
+      #     service_account: '${{ vars.gcp_service_account }}' # sourced from gh environment variable
+      #     workload_identity_provider: '${{ vars.gcp_workload_identity_provider }}'
+
```
Comment (florianow) [inline .github/workflows/build-workflow.yml:47]: do we need it when it is greyed out?

File: kit/foundation/docs/main.tf (hunk 1 above)
Comment (JohannesRudolph) [reply kit/foundation/docs/main.tf:160]: will come soon :-) sorry for comitting commented code, that's an artifact of porting over the current code from our internal staging environment that's already running on the docs v2 setup

File: foundations/likvid-prod/docs/vuepress/docs/README.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
+# Welcome to meshcloud Internal Cloud Foundation
+
+Welcome to meshcloud Cloud Foundation. Our team's job is to help you get your applications to the cloud. Please feel free to contact us on Slack!
+
```
Comment (florianow) [inline foundations/likvid-prod/docs/vuepress/docs/README.md:4]: Welcome to likvid Cloud Foundation :)

File: foundations/likvid-prod/docs/vuepress/docs/README.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+| Patrick Moore     | Platform Engineer |
+
+## Your way to the cloud
+
+The following requirements must be completed before you can start building and deploying your cloud resources
+
+<!-- TODO: Reword this to give more context and info. how to AWS account/Azure subscription? -->
+ 1. A Budget for provisioning cloud services. Please contact Nathalie Bermudez on Slack channel #flock-operations
+ 2. Access to [meshcloud-prod](https://panel.prod.meshcloud.io). See notion page for info on how to gain access [here](https://www.notion.so/meshcloud/Get-Access-to-meshcloud-prod-Panel-d6ef7478845342f699fb0d571f27b9dd)
+ 3. A dedicated meshcloud [Workspace](https://docs.meshcloud.io/docs/meshstack.how-to.get-started-with-meshstack.html#step-2-create-your-first-meshworkspace)
```
Comment (florianow) [inline foundations/likvid-prod/docs/vuepress/docs/README.md:25]: i guess that can out

File: kit/foundation/docs/template/vuepress/docs/README.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+
+The following requirements must be completed before you can start building and deploying your cloud resources
+
+<!-- TODO: Reword this to give more context and info. how to AWS account/Azure subscription? -->
+ 1. A Budget for provisioning cloud services. Please contact Nathalie Bermudez on Slack channel #flock-operations
+ 2. Access to [meshcloud-prod](https://panel.prod.meshcloud.io). See notion page for info on how to gain access [here](https://www.notion.so/meshcloud/Get-Access-to-meshcloud-prod-Panel-d6ef7478845342f699fb0d571f27b9dd)
+ 3. A dedicated meshcloud [Workspace](https://docs.meshcloud.io/docs/meshstack.how-to.get-started-with-meshstack.html#step-2-create-your-first-meshworkspace)
+
+After completing those steps, you can start integrating a meshPlatform into your meshStack. [Our integration documentation](https://docs.meshcloud.io/docs/meshstack.how-to.integrate-meshplatform.html) can guide you through this process. It has information on how to get started with the major public cloud provider Google, AWS, and Microsoft Azure as well as private cloud providers OpenShift and CloudFoundry
+
```
Comment (florianow) [inline kit/foundation/docs/template/vuepress/docs/README.md:28]: same here, infos from prod

**Review by florianow — APPROVED** [review]
LGTM some comments
